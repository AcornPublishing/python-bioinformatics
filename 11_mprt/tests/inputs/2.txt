P13473_LMP2_HUMAN
P42098_ZP3_PIG
P80069_A45K_MYCBO
Q13VE3
P20840_SAG1_YEAST
P01042_KNH_HUMAN
P07204_TRBM_HUMAN
Q7S432
A3DF24
Q6GEK4
P07585_PGS2_HUMAN
A9QYN2
Q5WFN0
Q9QSP4
