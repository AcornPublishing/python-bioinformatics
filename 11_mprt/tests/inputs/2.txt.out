P13473_LMP2_HUMAN
32 38 49 58 75 101 123 179 229 242 257 275 300 307 317 356
P42098_ZP3_PIG
124 146 179 271
P80069_A45K_MYCBO
7 161
Q13VE3
95
P20840_SAG1_YEAST
79 109 135 248 306 348 364 402 485 501 614
P01042_KNH_HUMAN
48 169 205 294
P07204_TRBM_HUMAN
47 115 116 382 409
Q7S432
173
A3DF24
178
P07585_PGS2_HUMAN
211 262 303
Q9QSP4
196 250 326 443
